public/proto/qf/QuickfeedServiceClientPb.js

```diff
@@ -14,127 +14,127 @@ var qf_types_pb = require("../qf/types_pb");
 var qf_requests_pb = require("../qf/requests_pb");
 var QuickFeedServiceClient = /** @class */ (function () {
     function QuickFeedServiceClient(hostname, credentials, options) {
-        this.methodInfoGetUser = new grpcWeb.MethodDescriptor('/qf.QuickFeedService/GetUser', grpcWeb.MethodType.UNARY, qf_requests_pb.Void, qf_types_pb.User, function (request) {
```

Comment:
@0xf8f8ff @JosteinLindhom Is this difference related to different versions of the `protoc-gen-grpc-web`??